common/changes/miwhea-fabric-core-6_2017-02-07-22-38.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Update to Fabric Core 6.0.0",
+      "type": "patch"
```

Comment:
What type is this? It doesn't add functionality or change the API, so I was leaning towards 'patch'. But if developers can use Fabric React to get a copy of Fabric Core, then this is a breaking change since it's a major version of Fabric Core.